broker/config/crds/deployment.servicefabrik.io_v1alpha1_dockers.yaml

```diff
@@ -35,6 +35,8 @@ spec:
               type: string
             state:
               type: string
+            actualState:
```

Comment:
It is not `actualState` right? Rather `appliedOptions`.

Reply:
yes, that is correct. Will modify.